spec/services/fetch_csv_services_failure_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'webmock/rspec'
+
+RSpec.describe Services::FetchCsvService, method: :call do
```

Comment:
Not very clear to me what is the benefit of splitting this test files in _failure / _success ?

Reply:
It's better to have tests splitted into multiple files because isolated tests reduce side-effects and allow better parallel execution with [parallel_tests](https://github.com/grosser/parallel_tests).

Reply:
VCR or stub_request from Webmock can be used as you've done so in other instances.

Reply:
> It's better to have tests splitted into multiple files because isolated tests reduce side-effects and allow better parallel execution with [parallel_tests](https://github.com/grosser/parallel_tests).

Hm, not convinced, if tests have side-effects are poorly written and should fix that, and parallel execution should work if you write tests in a single file. Is a lot easier to follow the tests if they are grouped together (also smaller things like IDE integration)